server/views/people/partials/personDetails.njk

```diff
@@ -22,7 +22,7 @@
         text: "Nationality"
       },
       value: {
-        text: person.nationality
+        text: person.nationality or 'Unknown'
```

Comment:
do we need this here if 'Unknown' has already been set in the javascript function?

Reply:
yes, this one is to populate the 'Confirm applicant details' page, the other one is to populate CYA and submitted application views